ports/esp32/esp32_common.cmake

```diff
@@ -196,5 +196,9 @@ foreach(comp ${IDF_COMPONENTS})
     micropy_gather_target_properties(${comp})
 endforeach()
 
+
+list(APPEND MICROPY_CPP_INC_EXTRA ${IDF_PATH}/components/riscv/include)
+list(APPEND MICROPY_CPP_INC_EXTRA ${IDF_PATH}/components/soc/soc/include)
```

Comment:
I'm expecting these lines to _not_ be required for real / merged support, but I've been having issues with the risk-v compiles and this is part of the WIP to get it going.